LBH.AdultSocialCare.Api/V1/Infrastructure/Entities/NursingCarePackage.cs

```diff
@@ -101,11 +102,16 @@ public class NursingCarePackage
         /// <summary>
         /// Gets or sets the Date Updated
         /// </summary>
-        public DateTime? DateUpdated { get; set; }
+        public DateTimeOffset? DateUpdated { get; set; }
```

Comment:
Not a change request, but this field can be attributed with `[DatabaseGenerated(DatabaseGeneratedOption.Identity)]` to receive a value on insert/update automatically